bin/jest.js

```diff
@@ -155,6 +161,12 @@ var argv = optimist
       );
     }
 
+    if (argv.watchExtensions && argv.watch === undefined) {
+      throw (
```

Comment:
The other ifs in checks only throws a string too, should I change them as well in this PR?